chart/values.yaml

```diff
@@ -475,9 +476,11 @@ loki-stack:
     persistence:
       # -- Enable persistence storage for the logs
       enabled: true
-      # -- StorageClass for Loki's centralised log storage
-      # empty storageClass implies cluster default storageClass & `manual` creates a static hostpath PV
-      storageClassName: ""
+      # -- StorageClass for Loki's centralised log storage. Options:-
+      # <p> - `"openebs-hostpath"` (default) - Will provision openebs local-pv hostpath volume. Needs `Values.openebs-localpv-provisioner.enabled = true`. <br>
+      # - `"manual"` - Will provision a hostpath PV on the same node. <br>
+      # - `""` (empty) - Will use the default StorageClass on the cluster. </p>
+      storageClassName: "openebs-hostpath"
```

Comment:
I suspect not but I'll ask anyway, we can't somehow set this from another variable right?

Reply:
We can change the name to whatever we want but we cannot use any other key value pair in the Values.

Reply:
I mean have something like
storageClassName: .Values.hostPath.storageClass